README.md

```diff
@@ -25,6 +25,32 @@ may eventually evolve into becoming a dynamic server.
 
 To get a local version of the site running, you'll need to have [git](http://git-scm.com/) and [node](http://nodejs.org/) installed on your local machine.
 
+For best results, use node version 0.12.7 (as of November 2015).  Verify your node version using:
+
+```
+node --version
+```
+
+If you need to install node, or switch to a different version of node, Node Version Manager (nvm) is a helpful tool.
+To use nvm for installing and managing different versions of node, first install prerequisite libraries using:
```

Comment:
I feel we should split these instructions out into linux/os x/windows instructions here instead of having users skim/copy/paste/frustrate when on other platforms. Thoughts, @PaulaPaul?